2fa/v1/openapi.yaml

```diff
@@ -19,10 +23,17 @@ security:
   - APIKeyHeader: []
 basePath: /
 tags:
-  - name: 2fa-application-service
-    description: 2FA Application Service
-  - name: 2fa-otp-service
-    description: 2fa Otp Service
+  - name: "Application Service"
+    description: |
+      The 2FA API has a part where the user can create different 2FA applications and separate his configuration.
+      tyntec stores these configurations for the user and the user can refer to the applicationId that points to 
+      the desired configuration that will be applied for the specific OTP delivery.
+
+      There is a default application for each user which contains some default values and an English language template.
+      This default application will be used in case no applicationId is specified in an OTP delivery request.
+      The user can change the default parameters of the default application as well.
+  - name: "OTP Service"
```

Comment:
This is somehow confusing when the section talks about 2FA, or?

Reply:
Yes, it was a little bit confusing. I rephrase those paragraphs. I guess now it is better and more understandable.